scripts/install-jstz-cli.sh

```diff
@@ -38,7 +39,13 @@ jstz_configure() {
         mkdir -p "$jstz_home"
         cat >"$jstz_home/config.json" << EOF
 {
-  "default_network": "dev"
+  "default_network": "dev",
+  "networks": {
+    "weeklynet": {
+      "octez_node_rpc_endpoint": "http://rpc.$network.teztnets.com",
+      "jstz_node_endpoint": "http://34.39.12.211:8933"
+    }
+  }
 }
 EOF
 
```

Comment:
where is the JSTZ_VERSION defined?

Reply:
It was moved to the `$version` variable -- it is not standard for all bash variables to be caps (just non-local variables)